DNAmicAnalysis.py

```diff
@@ -168,52 +195,62 @@ def main(cfg):
     unique_svcacct_domain_admins = db.exec_fromfile("data/sql/UniqueSvcDomainAdmins.sql", True, svc_array)
     unique_svcacct_domain_admins2 = db.exec_fromfile("data/sql/UniqueSvcDomainAdmins2.sql", True, svc_array)
 
-    unique_svcacct_domadm_usernames = []
-    if unique_svcacct_domain_admins:
-        for username in unique_svcacct_domain_admins:
-            unique_svcacct_domadm_usernames.append(username[0])
+    if unique_domain_admins and unique_svcacct_domain_admins and unique_svcacct_domain_admins2:
+        unique_svcacct_domadm_usernames = []
+        if unique_svcacct_domain_admins:
+            for username in unique_svcacct_domain_admins:
+                unique_svcacct_domadm_usernames.append(username[0])
 
-    unique_svcacct_domadm2_usernames = []
-    if unique_svcacct_domain_admins2:
-        for username in unique_svcacct_domain_admins2:
-            unique_svcacct_domadm2_usernames.append(username[0])
+        unique_svcacct_domadm2_usernames = []
+        if unique_svcacct_domain_admins2:
+            for username in unique_svcacct_domain_admins2:
+                unique_svcacct_domadm2_usernames.append(username[0])
+    else:
+        unique_domain_admins = False
+        unique_svcacct_domain_admins = []
+        unique_svcacct_domadm2_admins = []
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'unique_svcacct_domadm2_admins' is assigned to but never used](https://app.codacy.com/app/infamousjoeg/DNAmicAnalysis/pullRequest?prid=4577859)